tests/ci/README.md

```diff
@@ -52,13 +52,12 @@ CodeBuild|gcc 7|corretto 17|aarch|Ubuntu 20.04|FIPS/non-FIPS
 (macOS CI dimension is currently disabled, go to the Actions tab in the main repo to enable it when its ready.)
 
 
-### Dieharder & Overkill tests
+### Overkill tests
```

Comment:
looks like we've also removed the overkill tests?

Reply:
[The overkill tests](https://github.com/corretto/amazon-corretto-crypto-provider/blob/main/build.gradle#L568) task has been modified by removing dieharder from it. It is still there.